include/mcl_3dl/state_6dof.h

```diff
@@ -155,6 +155,34 @@ class State6DOF : public mcl_3dl::pf::ParticleBase<float>
   {
     rot_.normalize();
   }
+  size_t covDimension() const override
+  {
+    return 6;
+  }
+  float covElement(const State6DOF& e, const size_t& j, const size_t& k)
+  {
+    const State6DOF exp = e;
```

Comment:
Is it needed to copy?

Reply:
Removed in f3a12d71b0b0b92c8553515a6771acd84a292d8c.